plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -131,8 +131,8 @@ def test_jobs_configured_via_cmd(hydra_sweep_runner) -> None:
         assert returns["optimizer"] == "ax"
         assert len(returns) == 2
         best_parameters = returns["ax"]
-        assert math.isclose(best_parameters["quadratic_x"], -2.0, abs_tol=1e-4)
-        assert math.isclose(best_parameters["quadratic_y"], 2.0, abs_tol=1e-4)
+        assert math.isclose(best_parameters["quadratic_x"], -2, abs_tol=1e-4)
```

Comment:
2.0 was wrong?

Reply:
ohh I think it is looking this because at some point (while working on this PR) I changed the interpretation of interval and the value of output became some floating point number. Then when I updated the interpretation, I switched the value to 2 instead of 2.0. Reverting it.